submissions/Ant-C-tech/memory-pair-game/js/app.js

```diff
@@ -0,0 +1,278 @@
+'use strict'
+
+import hiragana from './hiragana.js'
+
+
+const BODY = document.querySelector('.body')
+const MAIN = document.querySelector('#main')
+const HEADER = document.querySelector('#header')
+const DESCRIPT = document.querySelector('#descript')
+
+const CARDS_NUMBER = 12;
+const ANIMATE_CSS_CLASS = 'animate__animated'
+const ANIMATE_DELAY = 'animate__delay-3s'
+const DESCRIPT_HIDE_ANIMATION = 'animate__zoomOut'
+const DESCRIPT_SHOW_ANIMATION = 'animate__zoomIn'
+const CARDS_HIDE_ANIMATION = 'animate__rotateOut'
+
+let TASK = []
+let attempts = 0
+    //Music
+const GAMEPLAY_AUDIO = new Audio('./audio/Japanese_Countryside.mp3')
+GAMEPLAY_AUDIO.loop = true
+GAMEPLAY_AUDIO.volume = 0.3
+
+
+initApp()
+
+
+function initApp() {
+    setTask()
+    const timeout = setTimeout(() => {
+        changeContent(createGreeting())
+        clearTimeout(timeout)
+    }, 1500);
+}
+
+function setTask() {
+    TASK.push(hiragana[_getRandomIntInclusive(0, hiragana.length - 1)])
+    while (TASK.length < CARDS_NUMBER / 2) {
```

Comment:
It's not a blocker. Just an advice for future. 
Try to design the logic of your app avoiding using the loop in loop. It will help you to increase the performance.